index.css

```diff
@@ -0,0 +1,276 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+}
+body {
+  font-size: 16px;
+}
+.logo__yellowColor {
+  color: #f8cb45;
+}
+.logo__greenColor {
+  color: #54b226;
+}
+.flexRow {
+  display: flex;
+  flex-direction: row;
```

Comment:
1. Default value of `flex-direction` is `row`, so we don't need to give the same value
2. We can add `align-items: center`, because you have used that many places